src/Rootfs.jl

```diff
@@ -497,8 +497,8 @@ function choose_shards(p::AbstractPlatform;
             preferred_llvm_version::VersionNumber = getversion(LLVM_builds[end]),
         )
 
-    # Our host platform is x86_64-linux-musl
-    host_platform = Platform("x86_64", "linux"; libc="musl")
+    # Our host platform is x86_64-linux-musl-cxx11
+    host_platform = default_host_platform
```

Comment:
Can we just replace usage of `host_platform` with `default_host_platform`?

Reply:
Good point, we aren't really allowing to use something else